### PR TITLE
Ruby 3.2.0 Compatibility

### PR DESCRIPTION
**Pull request by ruban-thilak**

### CODE REVIEW

**Review by darkleaf — CHANGES_REQUESTED** [review]
@ruban-thilak please add some info. Why do these changes add support for Ruby 3.2?

File: lowkiq.gemspec [hunk 1]
```diff
@@ -34,5 +34,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rspec", "~> 3.0"
   spec.add_development_dependency "rspec-mocks", "~> 3.8"
   spec.add_development_dependency "rack-test", "~> 1.1"
-  spec.add_development_dependency "pry-byebug", "~> 3.9.0"
+  spec.add_development_dependency "pry-byebug", "~> 3.10.1"
```
Comment (darkleaf) [inline lowkiq.gemspec]: I think we should use any version of `pry-byebug`.

Comment (ruban-thilak) [reply lowkiq.gemspec]: Checking it @darkleaf

Comment (ruban-thilak) [reply lowkiq.gemspec]: @darkleaf can we have `spec.add_development_dependency "pry-byebug", ">= 3.9.0"` ? because ruby 3.2.0 requires pry-byebug 3.10.

Comment (darkleaf) [reply lowkiq.gemspec]: add ruby 3.2 to the matrix

Comment (ruban-thilak) [reply lowkiq.gemspec]: done !

**Review by darkleaf — APPROVED** [review]

**Pull request merged**